docs/troubleshooting/new-connection.md

```diff
@@ -24,11 +22,24 @@ One workaround is to manually pull the latest version of every connector you'll
 
 If the above workaround does not fix your problem, please report it [here](https://github.com/airbytehq/airbyte/issues/1462) or in our [Slack](https://slack.airbyte.io).
 
-### **Connection refused errors when connecting to a local db**
+## Connection refused errors when connecting to a local db
 
 Depending on your Docker network configuration, you may not be able to connect to `localhost` or `127.0.0.1` directly.
 
 If you are running into connection refused errors when running Airbyte via Docker Compose on Mac, try using `host.docker.internal` as the host. On Linux, you may have to modify `docker-compose.yml` and add a host that maps to your local machine using [`extra_hosts`](https://docs.docker.com/compose/compose-file/compose-file-v3/#extra_hosts).
 
-### I don’t see a form when selecting a connector
+## I don’t see a form when selecting a connector
+
 We’ve had that issue once. (no spinner & 500 http error). We don’t know why. Resolution: try to stop airbyte (`docker-compose down`) & restart (`docker-compose up`)
+
+## Connection hangs when trying to run the discovery step
+
+You receive the error below when you tried to sync a database with a lot of tables (6000 or more).
```

Comment:
Since we are talking in terms of 'databases`, it's probably less confusing to stick to that terminology.